src/pkg/packager/create.go

```diff
@@ -576,6 +585,9 @@ func (p *Packager) loadDifferentialData() error {
 		return nil
 	}
 
+	// Save the fact that this is a differential build into the build data of the package
+	p.cfg.Pkg.Build.Differential = true
```

Comment:
This is being set twice and actually whatever value you set here will be overwritten based on later logic.

Reply:
I am removing it from the other location it's being set.